src/main/java/main.java

```diff
@@ -0,0 +1,7 @@
+import controller.LottoController;
+
+public class main {
+    public static void main(String[] args) {
+        LottoController.LottoAutoStart();
```

Comment:
```suggestion
        LottoController.lottoAutoStart();
```
이 부분도 카멜케이스에 맞게 수정하면 좋겠네요